dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScopeTest.kt

```diff
@@ -131,7 +135,8 @@ internal class RumActionScopeTest {
             TEST_INACTIVITY_MS,
             TEST_MAX_DURATION_MS,
             mockRumEventSourceProvider,
-            fakeAndroidInfoProvider
+            fakeAndroidInfoProvider,
+            true
```

Comment:
shouldn't we take the forged value?

```suggestion
            fakeTrackFrustrations
```

Reply:
We can. The issue is that we have several tests with code blocks similar to this: 
```
if (fakeType == RumActionType.TAP) {
        hasFrustration(ActionEvent.Type.ERROR_TAP)
    } else {
        hasNoFrustration()
    }
}
```
We can either keep this value as true and not alter them, or use a forged value and change the if condition to something like `fakeTrackFrustraions && fakeType == RumActionType.TAP`.

What do you think is best?

Reply:
ok, makes sense. Let's keep it as `true` by default then.